utils/build-windows-toolchain.bat

```diff
@@ -123,13 +123,8 @@ setlocal enableextensions enabledelayedexpansion
 git config --global core.symlink true
 
 :: FIXME(compnerd) avoid the fresh clone
-rd /s /q zlib libxml2 sqlite icu curl
-
-git clone --quiet --no-tags --depth 1 --branch v1.2.11 https://github.com/madler/zlib
-git clone --quiet --no-tags --depth 1 --branch v2.9.12 https://github.com/gnome/libxml2
-git clone --quiet --no-tags --depth 1 --branch version-3.36.0 https://github.com/sqlite/sqlite
+rd /s /q icu
 git clone --quiet --no-tags --depth 1 --branch maint/maint-69 https://github.com/unicode-org/icu
```

Comment:
This does not look right, we are changing the version of the ICU.

Reply:
This is not changing the version of ICU, ICU 69.1 is what Windows has used since 5.3.

Reply:
Another point, It's deleting local copy of the icu. This would be bad if an engineer had a ICU changes locally, they will lose the work.

Reply:
Now with @parkera @iCharlesHu feedback, we can drop the clone of icu from here.